pkg/compact/compact.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/tsdb"
 	"github.com/prometheus/tsdb/labels"
+
```

Comment:
make format needed

Reply:
The formatter didn't seem to actually remove this extra line but I did it manually anyway.